rhinosystem/gtk/upgrade.ui

```diff
@@ -31,12 +31,12 @@
         <property name="valign">fill</property>
         <property name="halign">center</property>
         <property name="hexpand">true</property>
-	<property name="vexpand">true</property>
-	<property name="margin-bottom">20</property>
-	<child>
+	      <property name="vexpand">true</property>
+	      <property name="margin-bottom">20</property>
+	      <child>
           <object class="GtkButton" id="quitButton">
             <property name="label">Exit Application</property>
-	    <style>
+	          <style>
```

Comment:
Please revert the indentation changes here.

Reply:
I didn't catch that somehow. One moment!

Reply:
It looks perfectly fine for me on VS code so idk...

Reply:
I "resaved" it, hopefully it's correct now.

Reply:
It is not
please check your vscode config and make sure that it sets tabs to spaces

Reply:
![image](https://github.com/rhino-linux/rhino-system/assets/95766563/c19076a7-56b2-457c-985f-a38601a8a2ef)
This is what it looks like to me...

Reply:
As I said, check your vscode config and make sure tabs are set to spaces with width 2 in this case.
You can also just open the file in a different text editor and fix it there.